R/score.R

```diff
@@ -260,6 +260,16 @@ score.scoringutils_quantile <- function(data, metrics = metrics_quantile, ...) {
   return(data[])
 }
 
+
+#' Helper Function To Apply a List Of Functions To a Data Table of Forecasts
```

Comment:
is it a list of functions or a list of metrics (or whatever name we exhaustion marched to?)